build.proj

```diff
@@ -65,7 +65,23 @@
 
   <Import Project="dir.targets" />
   <Import Project="dir.traversal.targets" />
-  <Import Project="$(ToolsDir)clean.targets" />
+  <Import Project="$(ToolsDir)clean.targets" />  
+ 
+  <!-- 
+    This is to initialize the ProjectJsonFiles ItemGroup with the template files during the update dependencies target
+    only to prevent ValidateExactRestore from failing on just the regular project.json files.
+  -->
+  <Target Name="IncludeOnlyTemplateFilesForUpdate" BeforeTargets="UpdateDependencies">
+    <ItemGroup>
+      <ProjectJsonFiles Remove="*" />
```

Comment:
@weshaggard I thought it might interfere with the update pr but I see that project.json are included in the gitignore. I'll remove the Remove statement. 

I still think the target may be needed unless you'd prefer a condition on the ItemGroup.